Samples/HelloWorld/SiloTests/ClientConfigurationForTesting.xml

```diff
@@ -1,13 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<ClientConfiguration xmlns="urn:orleans">
-  <Gateway Address="localhost" Port="40000"/>
-  <!-- To turn tracing off, set DefaultTraceLevel="Off" and have no overrides.
-    For the trace log file name, {0} is replaced by "Client" and {1} is the current time. -->
-  <Tracing DefaultTraceLevel="Info" TraceToConsole="false" TraceToFile="{0}-{1}.log" BulkMessageLimit="1000">
-    <TraceLevelOverride LogPrefix="Runtime" TraceLevel="Info" />
-    <TraceLevelOverride LogPrefix="Application" TraceLevel="Info" />
-    <TraceLevelOverride LogPrefix="AssemblyLoader" TraceLevel="Warning" />
-  </Tracing>
-  <Statistics MetricsTableWriteInterval="300s" PerfCounterWriteInterval="30s" LogWriteInterval="300s" WriteLogStatisticsToTable="true" StatisticsCollectionLevel="Info"/>
-  <Messaging ResponseTimeout="30s" ClientSenderBuckets="8192" MaxResendCount="0"/>
-</ClientConfiguration>
```

Comment:
we are using programmatic config since the last PR we merged. This was a remnant of the old TestingSiloHost usage, but no longer being used.